src/net.h

```diff
@@ -640,80 +640,80 @@ class CNode
     friend class CConnman;
 public:
     // socket
-    std::atomic<ServiceFlags> nServices;
+    std::atomic<ServiceFlags> nServices{NODE_NONE};
     SOCKET hSocket GUARDED_BY(cs_hSocket);
-    size_t nSendSize; // total size of all vSendMsg entries
-    size_t nSendOffset; // offset inside the first vSendMsg already sent
-    uint64_t nSendBytes GUARDED_BY(cs_vSend);
+    size_t nSendSize{0}; // total size of all vSendMsg entries
+    size_t nSendOffset{0}; // offset inside the first vSendMsg already sent
+    uint64_t nSendBytes GUARDED_BY(cs_vSend){0};
```

Comment:
What about ...

```
uint64_t nSendBytes = 0 GUARDED_BY(cs_vSend);
```

... to make it clear that `0` has nothing to do with `GUARDED_BY(cs_vSend)`?

Apples for the other member variables with `GUARDED_BY(…)`:s.

Reply:
I prefer consistency, and maybe a space around the annotation would look better:
```cpp
uint64_t nSendBytes GUARDED_BY(cs_vSend) {0};
```